app/Http/Controllers/API/v1/Student/StudentActivitiesController.php

```diff
@@ -6,6 +6,8 @@
 use Illuminate\Http\Request;
 use App\Models\User;
 use Spatie\Activitylog\Models\Activity;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Auth;
```

Comment:
```suggestion
use Illuminate\Support\Facades\Auth;
use Illuminate\Support\Facades\DB;
```
Please arrange alphabetically